ENH Adds missing value support to OneHotEncoder (#17317)
scikit-learn/scikit-learn: doc/whats_new/v0.24.rst

:mod:`sklearn.preprocessing`
............................

- |Feature| :class:`preprocessing.OneHotEncoder` now supports missing

>   values by treating them as its own category. :pr:`17317` by `Thomas Fan`_.

Reviewer: ```suggestion
  values by treating them as a category. :pr:`17317` by `Thomas Fan`_.
```
